static/index.html

```diff
@@ -87,8 +88,46 @@ <h4>Gold Sponsors</h4>
       </div>
     </section>
     <footer>
-      <div class="container">
-        <div class="row">
+      <div class="footer-top">
+        <div class="container-fluid">
+          <div class="row">
+              <div class="col-lg-3">
+                <div class="drupalcebu_header">
+                  <h3>Drupal in Cebu</h3>
+                  <p>This camp is brought to you by <a href="https://wwww.facebook.com/groups/drupalcebu" target="_blank">Drupal Cebu User Group</a></p>
+                  <p>Access the source code of this website on Github. This website is all open source</p>
+                  <p>Drupal meetups every last Wednesday of the month, find us on <a href="https://www.meetup.com/drupal-ph/" target="_blank">meetup.com</a></p>
+                  <p class="button-link">
+                    <a class="button-link text-display-large" href="https://www.meetup.com/drupal-ph/" target="_blank">Drupal Monthly Meetups</a>
+                  </p>
+                </div>
+              </div>
+              <div class="col-lg-3">
+                <div class="drupaltwitter_header">
+                  <h3>Drupalcebu on Twitter</h3>
+                  <p> <i class="fa fa-twitter"></i><a href="https://twitter.com/drupalcampcebu" target="_blank">@DrupalCampCebu</a> Announcing Drupalcamp's keynote: <a href="https://twitter.com/johnalbin" target="_blank">@JohnAlbin !</a> More information on <a href="https://2018.drupalcebu.org">https://2018.drupalcebu.org</a></p>
+                  <p> <i class="fa fa-twitter"></i><a href="https://twitter.com/drupalcampcebu" target="_blank">@DrupalCampCebu</a> Planning a trip to South East Asia in November? Join the fun in Cebu! #drupal #drupical #travel</p>
```

Comment:
Please update the content from: https://twitter.com/drupalcampcebu
Exclude retweets.